lua/example_of_dynamic_config_request.lua

```diff
@@ -5,4 +5,43 @@
 -- Functions to process the GT policies.
 -- ......
 
-return "A default message from the dynamic configuration block!"
+local dylib = require("dylib")
+local ffi = require("ffi")
+
+local dyc = gatekeeper.c.get_dy_conf()
+
+-- Setup Grantor Prefixes.
+local ret = dylib.c.add_fib_entry_lock("128.197.40.100/24",
+	dylib.c.GK_FWD_GATEWAY_BACK_NET, "10.0.1.254", dyc.gk)
+if ret < 0 then
+	return "gk: failed to add an FIB entry\n"
+end
+
+-- Setup Prefixes that need to forward to the back network.
+ret = dylib.c.add_fib_entry_lock("187.73.40.0/30",
+	dylib.c.GK_FWD_GRANTOR, "128.197.40.100", dyc.gk)
+if ret < 0 then
+	return "gk: failed to add an FIB entry\n"
+end
+
+ret = dylib.c.del_fib_entry_lock("187.73.40.0/30", dyc.gk)
+if ret < 0 then
+	return "gk: failed to delete an FIB entry\n"
+end
+
+ret = dylib.c.del_fib_entry_lock("128.197.40.100/32", dyc.gk)
+if ret < 0 then
+	return "gk: failed to delete an FIB entry\n"
+end
+
+ret = dylib.c.del_fib_entry_lock("128.197.40.100/24", dyc.gk)
+if ret < 0 then
+	return "gk: failed to delete an FIB entry\n"
+end
+
+ret = dylib.c.del_fib_entry_lock("10.0.1.254/32", dyc.gk)
+if ret < 0 then
+	return "gk: failed to delete an FIB entry\n"
+end
+
+return "gk: successfully processed all the FIB entries\n"
```

Comment:
I see now, I wasn't paying close enough attention to the prefixes lengths on the end. Makes sense, thank you!